extensions/vertx-http/dev-ui-resources/src/main/resources/dev-ui/qwc/qwc-extensions.js

```diff
@@ -153,10 +183,35 @@ export class QwcExtensions extends observeState(LitElement) {
                     providesCapabilities="${extension.providesCapabilities}"
                     extensionDependencies="${extension.extensionDependencies}">
                     <div class="card-content" slot="content">
+                        <span class="float-right badge badge-warning">EXPERIMENTAL</span>
```

Comment:
You can probably remove this duplication (in the else) and just here call a method that render the badge if needed